### PR TITLE
Adding Locale / Language Switcher component with flags (optional).

### PR DESCRIPTION
**Pull request by jpaddeo**
I added a Local or Language Switcher component in Header next to Profile Info or Login Button (if not session).

It looks like this:
* Without flags emoji.
image

*  With flags emoji:
image

**UPDATE**: the images are old. New changes made with .env variable to control the "layout" of selector (text, flag or both). And removing USA flag for EN.

### CODE REVIEW

**Comment by edgarguitarist** [comment]
I have a question... If you only have 2 languages, why use a Select?  When can it be a toggle button?  I asked because I've seen it a lot on other sites and I don't know if it's because other languages ​​were added later or for some other reason! I want to learn! 🙌🏽

**Comment by jpaddeo** [comment]
> I have a question... If you only have 2 languages, why use a Select? When can it be a toggle button? I asked because I've seen it a lot on other sites and I don't know if it's because other languages ​​were added later or for some other reason! I want to learn! 🙌🏽

Hi @edgarguitarist ! Yes, as you said, for now we only have two languages but the idea, if I'm not bad (@midudev pls validate it), is to have more than two. The idea is that de site was international and can be used by all worlwide.

File: components/LocaleSwitcher.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import { getFlagLocale } from '../helpers/flags'
+
+interface LocaleSwitcherProps {
+  useFlags?: boolean
+}
+
+function LocaleSwitcher({ useFlags = false }: LocaleSwitcherProps) {
+  const { locales, locale } = useRouter()
+  const router = useRouter()
+  const { pathname, asPath, query } = router
```
Comment (kevinzunigacuellar) [inline components/LocaleSwitcher.tsx]: Creo que esta linea puede ser combinada con la linea 10 ya que ambas deestructuran el mismo objeto.

Comment (jpaddeo) [reply components/LocaleSwitcher.tsx]: Kevin, you're right! I fixed it on new commits (just merged). Thanks for the comment!

**Pull request merged**